Set-SslSecurity.ps1

```diff
@@ -101,11 +101,15 @@ process {
     } until($response.StatusCode -eq '200')
     Write-Host "Nexus is ready!"
 
-    # Update Repository URI
+    # Update Repository URI and API key
     choco source remove --name="'ChocolateyInternal'"
     $RepositoryUrl = "https://${SubjectWithoutCn}:8443/repository/ChocolateyInternal/"
     choco source add --name="'ChocolateyInternal'" --source="'$RepositoryUrl'" --priority=1
-    choco apikey --source="'$RepositoryUrl'" --apikey $NuGetApiKey
+    $ChocoArgs = @{
```

Comment:
```suggestion
$chocoArgs = @('apikey',"--source='$RepositoryUrl'","--api-key='$NuGetApiKey'")
& choco @chocoArgs
```

Reply:
This is the array syntax I was referring too. Hashtable is fine, but per our conversation outside github, this was the syntax I was referring too.